starter-code/index.html

```diff
@@ -1,31 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" href="./css/main.css">
   <title>Instagram</title>
 </head>
-<body>
-  Instagram
+<body class ="css3gradient">
+<div class="divForPhones">
 
-  Capture and Share the World's Moments
+    <img src="/Users/ivanjorge/Prework/code/labs/lab-css-instagram-clone/starter-code/images/phones.png" alt ="" class="phones">
```

Comment:
Remember to use relative paths when linking images in your HTML. The way your file structure is set up is as follows:

Main Directory [Contains index.html and the images folder]
Images folder [Contains all of your images]

Your images currently don't show up when the HTML file is opened because the browser doesn't know about any files or folders outside of the main directory: this includes 'Users', 'ivanjorge', 'Prework', etc. 

The correct path to use is 'images/phones.png'. This tells the browser, which starts in the main directory, to go into the images folder, and look for phone.png.